litr-demo/src/main/java/com/linkedin/android/litr/demo/data/AudioTarget.java

```diff
@@ -22,6 +23,7 @@ public AudioTarget() {
     @Override
     public void onCheckedChanged(CompoundButton button, boolean checked) {
         shouldTranscodeAudio = checked;
+        shouldKeepTrack = checked;
```

Comment:
is it confusing? e.g. when shouldTranscodeAudio is false, then shouldKeepTrack is false